modules/aws/dra-admin/variables.tf

```diff
@@ -28,7 +28,7 @@ variable "key_pair" {
 variable "dra_version" {
   description = "The DRA version to install"
```

Comment:
I suggest adding to the description - Supported versions are 4.11.0.10.0.7 and up. Similar to Sonar.